sct-commons/src/test/java/org/lfenergy/compas/sct/commons/scl/PrivateServiceTest.java

```diff
@@ -151,6 +151,41 @@ void getCompasPrivate_should_throw_exception() {
             ScdException.class);
     }
 
+    @Test
+    void getCompasPrivate_on_base_element_should_return_private_value() {
```

Comment:
bad test method names, try to explain the cases and the cause which leads to return type

Reply:
The method name seems clear to me.
Do you have a suggestion for a better name ?

Reply:
`getCompasPrivate_on_base_element_should_return_private_value_when_present`